Changelog.md

```diff
@@ -7,13 +7,15 @@ enforce some safety features. The most important change is
 Breaking Changes:
 
  * Source files have to specify the compiler version they are
-   compatible with using e.g. `pragma solidity ^0.4.0;` or
-   `pragma solidity >=0.4.0 <0.4.8;`
+   compatible with using e.g. ``pragma solidity ^0.4.0;`` or
+   ``pragma solidity >=0.4.0 <0.4.8;``
+ * Functions that want to receive Ether have to specify the
+   new ``payable`` modifier (otherwise they throw).
  * Contracts that want to receive Ether have to implement a fallback
-   function (contracts now throw if no fallback function is defined
-   and no function matches the signature).
- * Failing contract creation through "new" throws now.
- * Throw on division / modulus by zero
+   function with the payable modifier (contracts now throw if no fallback
+   function is defined and no function matches the signature).
```

Comment:
*no fallback functions is defined*, *the one defined is not payable* or *no function matches*